src/Command/ResetPasswordCommand.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Bolt\Command;
+
+use Bolt\Common\Str;
+use Bolt\Entity\User;
+use Bolt\Repository\UserRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\InvalidArgumentException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+use Symfony\Component\Validator\Validator\ValidatorInterface;
+
+/**
+ * Resets password for the user with the specified username.
+ *
+ * @package App\Command
+ */
+class ResetPasswordCommand extends Command
+{
+    protected static $defaultName = 'bolt:reset-password';
+
+    /** @var EntityManagerInterface */
+    private $entityManager;
+
+    /** @var UserPasswordEncoderInterface */
+    private $passwordEncoder;
+
+    /** @var ValidatorInterface */
+    private $validator;
+
+    /** @var UserRepository */
+    private $userRepository;
+
+    public function __construct(EntityManagerInterface $em, UserPasswordEncoderInterface $encoder,
+                                UserRepository $userRepository, ValidatorInterface $validator)
+    {
+        parent::__construct();
+
+        $this->entityManager = $em;
+        $this->passwordEncoder = $encoder;
+        $this->validator = $validator;
+        $this->userRepository = $userRepository;
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setDescription('Reset password for a user.')
+            ->addArgument('username', InputArgument::REQUIRED, 'User name')
```

Comment:
I'd say change 'User name' to 'Username of the user'

Reply:
Yes, that's more clear.